CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [13.1.0]
+### Fixed
+- revert error handling of supportsInterface method to general catch with no rethrow ([#530](https://github.com/MetaMask/controllers/pull/530))
```

Comment:
Since this is a bug fix for a bug that was introduced in this version, we can omit it from the changelog. Listing this might mislead library consumers into thinking this bug existed in the previous release.

It might be good to list this under the "Add ERC721" change entry, as part of the implementation. We can list multiple PRs for a single change.

Reply:
Thinking something like this: 
```
## Added
- Add ERC721 detection to TokensController ([#524](https://github.com/MetaMask/controllers/pull/524))
  - Revert error handling of supportsInterface method to general catch with no rethrow ([#530](https://github.com/MetaMask/controllers/pull/530))
```
or like this:
```
## Added
- Add ERC721 detection to TokensController ([#524](https://github.com/MetaMask/controllers/pull/524)), ([#530](https://github.com/MetaMask/controllers/pull/530))
```

Reply:
@Gudahtt ^

Reply:
I like how the second one looks, of those two options. Sub-bullets are OK to use in general, but with that first example it's a little confusing what is being referred to imo.